lib/discourse_id.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module DiscourseId
+  DEFAULT_PROVIDER_URL = "https://id.discourse.com"
+
+  def self.provider_url
+    SiteSetting.discourse_id_provider_url.presence || DEFAULT_PROVIDER_URL
+  end
+
+  def self.masked_client_id
```

Comment:
I think this is not needed now in the current version. We're not showing the masked ID in the UI (unless we're logging it?)